doc/enterprise-shield/o_enterprise_shield_checklist.md

```diff
@@ -0,0 +1,55 @@
+
+Traditionally, web architectures require that you open inbound ports to allow connectivity to internal systems and services in your trusted network. As with any web architecture, a typical KAAZING Gateway configuration (without Enterprise Shield™) must open ports to allow TCP, HTTP, or WebSocket connectivity through a firewall, as shown in the following figure:
+
+![Gateway Topology with Ports Open to the Trusted Network](../images/f-dmz-trusted-top.png)
+
+**Figure 1: A KAAZING Gateway Topology with Ports Open to the Trusted Network**
+
+Using Enterprise Shield™, you can close all inbound ports in your firewall, thus closing the entry points available to untrusted users and eliminating your attack surface vulnerability.
+
+To implement Enterprise Shield™, you configure a Gateway in the DMZ, which receives a reverse connection from within the trusted network. With this architecture, a client can talk to the back-end service or message broker through a firewall. Another benefit of Enterprise Shield™ is that your architecture remains valid, without requiring changes. For example, neither the client nor the back-end service or message broker are aware of the reverse connection because it is completely transparent to the rest of the architecture. Clients that are outside the firewall connect as usual to the DMZ Gateway.
+
+![Simple Topology Showing a Reverse Connection](../images/f-dmz-trustednetwork-860-02.png)
+
+**Figure 2: Closed Inbound Ports in an Enterprise Shield™ Topology**
+
+Adding Enterprise Shield™ to your architecture requires only a few simple modifications. Instead of a single Gateway in the DMZ, you add another Gateway to your trusted network. Then, you only need to make a few changes to the two Gateway configurations to reverse the connection. Other parts of the architecture, such as the client and the back-end service or message broker, observe no apparent differences between a configuration with Enterprise Shield™ or one without, making the reverse connection completely transparent to the endpoints of the configuration.
+
+With this architecture in place, you can close all inbound ports of your firewall, thus providing maximum security and zero attack vectors for malicious users seeking to exploit ports in your firewall.
+
+See [Common Enterprise Shield™ Use Cases](c_enterprise_shield_use_cases.md) to gain more insight into how you might use Enterprise Shield™. This topic describes topologies that vary depending on the level of security, such as configuring Enterprise Shield™ on a single pair of Gateways, on Gateways that are clustered for high availability, whether network transports are used, and so on.
```

Comment:
> This topic describes topologies that vary depending on the level of security, such as configuring Enterprise Shield™ on a single pair of Gateways, on Gateways that are clustered for high availability, whether network transports are used, and so on.

I think this sentence is too long. How about something simpler like, "This topic describes the various Enterprise Shield configurations you can deploy"?